.github/workflows/main.yml

```diff
@@ -118,6 +118,37 @@ jobs:
           files: /tmp/rekor-merged.cov,/tmp/pkg-rekor-merged.cov
           flags: e2etests
 
+  backfill:
+    runs-on: ubuntu-latest
+    needs: build
+
+    steps:
+      - uses: actions/checkout@1d96c772d19495a3b5c517cd2bc0cb401ea0529f # v4.1.3
+      - name: Extract version of Go to use
+        run: echo "GOVERSION=$(cat Dockerfile|grep golang | awk ' { print $2 } ' | cut -d '@' -f 1 | cut -d ':' -f 2 | uniq)" >> $GITHUB_ENV
+      - uses: actions/setup-go@0c52d547c9bc32b1aa3301fd7a9cb496313a4491 # v5.0.0
+        with:
+          go-version: ${{ env.GOVERSION }}
```

Comment:
Can we use `go-version-file: go.mod` instead? Dockerfile and go.mod should stay in sync.

Reply:
I can, but that would make this job inconsistent with every other workflow in rekor, fulcio, and cosign, for example line 89 above. Do you want this changed everywhere?

Reply:
Let's keep it consistent, we can change it in a follow up.